tenant-file/portal-app/src/component/home.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { ReactComponent as Logo } from "../images/logo.svg";
+import styled from 'styled-components';
+
+const StyledHeader = styled.div`
+  font-size: 100px;
+  color: ${(props) => props.theme.darkSecondary};
+  font-weight: 700;
+  text-align: center;
+
+  span {
+    align-text: center;
+  }  
+`;
+
+const Home: React.FC = () => (
+    // App logo and title
+    // <AppLogoAndTitle />
+    <StyledHeader>
+      <span>Home</span>
+      <Logo />
```

Comment:
Can you actually use this image? Just so it appears more "official" http://bastaaustin.org/images/casa%20text_mustard.png?crc=3890606587

Reply:
@ryanrightmer Done! Let me know what you think now